src/agency.js

```diff
@@ -1,8 +1,50 @@
-class Agency {
-  constructor() {
+import Finder from './Finder';
+import moment from 'moment'
```

Comment:
Smart to work this out with moment already installed...should save you some time with these methods in the long run.